src/System.Diagnostics.StackTrace/src/System.Diagnostics.StackTrace.sln

```diff
@@ -0,0 +1,34 @@
+﻿
```

Comment:
Can you just delete this SLN?  We usually put slns up a level and they include everything.